app/jobs/send_purge_warnings_job.rb

```diff
@@ -9,13 +9,13 @@ def perform(*_args)
     not_sent = []
 
     # Get admin users
-    recipients = User.with_any_role(:admin)
+    recipients = User.where(role: Roles::ADMIN) | User.where(role: Roles::SUPER_USER)
```

Comment:
So you just should be doing a normal `||` or operator here, the single pipe avoids lazy evaluation which we don't need.

Reply:
Actually sorry misread this line at first (didn't see it was OR-ing relations) - what you should do here is something like:
```
User.where(role: [Roles::ADMIN, Roles::SUPER_USER])
```

Because this will find users have a role that matches either of those values.
https://apidock.com/rails/ActiveRecord/QueryMethods/where
which will make a query for all records that have either of those Roles.